02week/nasa/index.html

```diff
@@ -2,37 +2,36 @@
 <html lang="en">
 
 <head>
-    <title>NASA</title>
-    <link rel="stylesheet" href="style.css">
+  <title>NASA</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <main>
-        <h1>NASA</h1>
-    </main>
-    <footer>
-        <div class="cheese">
-            <h3 class="header">Social Media</h3><br>
-            <p>
-                Twitter - @nasa<br> Facebook - fb.com/nasa<br> Instagram - @nasa<br>
-                YouTube - youtube.com/NASA<br>
-            </p>
-        </div>
-        <div class="cheese">
-            <h3 class="header">OFFICE INFO</h3><br>
-            <p>
-                Public Communications Office<br> NASA Headquarters<br> 300 E. Street
-                SW, Suite 5R30<br> Washington, DC 20546<br> (202)358-0001 -
-                (Office)<br> (202)358-4338 - (Fax)<br>
-            </p>
-        </div>
-        <div class="cheese">
-            <h3 class="header">CONTACT US</h3><br>
-            <p>
-                Name<br> Message<br>
-            </p>
-        </div>
-    </footer>
+  <main>
```

Comment:
Hi Allison, 

I commend your efforts to go ahead and work with the grid system. I know you are reworking this, but I wanted to go ahead and provide some comments that may be useful.